pkg/k8s/apis/cilium.io/utils/utils.go

```diff
@@ -100,8 +100,15 @@ func getEndpointSelector(namespace string, labelSelector *slim_metav1.LabelSelec
 	return es
 }
 
+func matchesPodInit(epSelector api.EndpointSelector) bool {
+	if epSelector.LabelSelector == nil {
+		return false
+	}
+	return epSelector.HasKey(podInitLbl)
+}
+
```

Comment:
Nit: generally in Golang we define the most important functions closer to the top of the file, and the less important (and in this case, a helper function) closer to the bottom, near the function that uses them. This way when someone wants to read through this file, they'll start with the most important functions and gather the "real" context of the file, rather than scrolling through "useless" helpers. :smile: